### ISSUE TITLE
Show simplified domain statuses to the registrant

### ISSUE THREAD

**Issue by Katherine-Osos** [cisagov/getgov#929]
### Story

As a domain manager
I want to know the status of my domain in simple language
so that I know if any action is needed or if any functions are limited

### Acceptance Criteria

- [x] Domains table on the dashboard shows a user-friendly domain status
- [ ] Show the domain status on the "Domain Overview" page
- [ ] For Domain statuses Deleted and Hold, change manage link to View with eye icon [(Figma)](https://www.figma.com/file/aJbuDLJNNikqQObTuCNKQa/GetGov_Designs_ECS-Truss?type=design&node-id=428%3A11621&mode=design&t=eottghQ6NoX98F6q-1)

### Additional Context
**BACKGROUND**
In general, EPP / domain statuses are not easily understandable to most users; therefore, we need to ensure we're showing "user-friendly" versions of the domain status. 

User-friendly statuses include: 
- _Note:_ "Unknown" _Domain status shows as_ "DNS needed"
- DNS needed 
- Ready
- On hold
- Deleted

Refer to[ status diagram Miro](https://miro.com/app/board/uXjVMuqbLOk=/?moveToWidget=3458764561795634398&cot=14)

**DOMAINS TABLE**
Currently, the approved Domains table displays the application status. But, because the application process is complete, we should update this to show the _domain_ status. 

Reference [Figma mock up](https://www.figma.com/file/aJbuDLJNNikqQObTuCNKQa/GetGov_Designs_ECS-Truss?type=design&node-id=428-25637&mode=design&t=VBfj3tkkg2p3f8UT-0)

**DOMAIN OVERVIEW**
Currently, we do not show the  domain status when viewing the "Manage Domains" pages. The "Manage Domains" pages can be accessed by clicking the "Manage" button next to an approved domain. 

The first page is the "Domain Overview." Add stylized message to the top of that page to indicate the user-friendly domain status.

Reference [Figma mock up](https://www.figma.com/file/aJbuDLJNNikqQObTuCNKQa/GetGov_Designs_ECS-Truss?type=design&node-id=4829-39693&mode=design&t=VBfj3tkkg2p3f8UT-0)

### Issue Links

Depends on: Domain Status presence for testing

**Comment by kitsushadow** [comment]
Could be reduced further by removing the icon pieces into a separate ticket.
